src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2087,7 +2087,7 @@ private function registerHttpClientRetry(array $retryOptions, string $name, Cont
             $retryServiceId = $name.'.retry.decider';
             $retryDefinition = new ChildDefinition('http_client.retry.abstract_httpstatuscode_decider');
             $retryDefinition
-                ->replaceArgument(0, $retryOptions['http_codes']);
+                ->replaceArgument(0, $retryOptions['http_codes'])
```

Comment:
How was this working before?..

Reply:
hmmm: it worked before. This is a bug. reverted